bcipy/gui/mode/RSVPKeyboard.py

```diff
@@ -15,7 +18,12 @@
 class RSVPKeyboard(BCIGui):
     """GUI for launching the RSVP tasks."""
     event_started = False
-    PARAMETER_LOCATION = 'bcipy/parameters/parameters.json'
+
+    def __init__(self, title, size, background_color, default_param_location='bcipy/parameters/parameters.json',
+                 params_queue=None):
+        super(RSVPKeyboard, self).__init__(title, size, background_color)
+        self.PARAMETER_LOCATION = default_param_location
+        self.params_queue = params_queue
```

Comment:
Just a clarification question, does params_queue meant for multiple .json params file locations to be stored and accessed?

Reply:
When a parameter JSON file is loaded from a non-default location, params_queue is used by the parameter editing form to send the new parameter location back to the main RSVPKeyboard GUI.